api/notd/api/models_v1.py

```diff
@@ -212,3 +212,22 @@ class ApiGmCollectionRow(BaseModel):
 class ApiLatestAccountGm(BaseModel):
     accountGm: ApiAccountGm
     accountCollectionGms: List[ApiAccountCollectionGm]
+
+
+class ApiCollectionOverlap(BaseModel):
+    collectionOverlapId: int
+    createdDate: datetime.datetime
+    updatedDate: datetime.datetime
+    registryAddress: str
+    galleryAddress: str
```

Comment:
note that you need to update these names everywhere accordingly with the comment from the db ye